container/features/src/main/resources/features-sentinel.xml

```diff
@@ -235,12 +235,27 @@
         <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.cassandra.newts-impl/${project.version}</bundle>
     </feature>
 
-    <feature name="sentinel-kvstore-cassandra"
-             description="OpenNMS :: Features :: Distributed :: Key Value Store :: Cassandra"
+    <feature name="sentinel-blobstore-cassandra"
+             description="OpenNMS :: Features :: Distributed :: Key Value Store :: Blob :: Cassandra"
              version="${project.version}">
         <feature>sentinel-newts-cassandra</feature>
-        <feature>opennms-kvstore-shell</feature>
+        <feature>opennms-blobstore-shell</feature>
         <feature>sentinel-kvstore-api</feature>
-        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.kv-store.cassandra/${project.version}</bundle>
+        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.kv-store.blob.cassandra/${project.version}</bundle>
+    </feature>
+
+    <feature name="sentinel-kvstore-postgres-shared"
```

Comment:
This feature won't resolve correctly if installed alone.

Since it only contains a single bundle, I would just move the bundle directly into the `sentinel-jsonstore-postgres` feature.